package.json

```diff
@@ -14,28 +14,28 @@
   "author": "elastic.io GmbH",
   "license": "BSD-2-Clause",
   "dependencies": {
-    "@elastic.io/component-commons-library": "0.0.6",
-    "@elastic.io/oih-standard-library": "1.0.1",
+    "@elastic.io/component-commons-library": "1.1.2",
+    "@elastic.io/oih-standard-library": "1.0.3",
```

Comment:
are you sure that all library changes should be used in this component?

Reply:
Changes were reverted